doc/modules/neural_networks_supervised.rst

```diff
@@ -0,0 +1,255 @@
+.. _neural_network:
+
+==================================
+Neural network models (supervised)
+==================================
+
+.. currentmodule:: sklearn.neural_network
+
+
+.. _elm:
+
+Extreme Learning Machines
+=========================
+
+**Extreme Learning Machine (ELM)** is a supervised nonlinear learning 
+algorithm that takes the following consecutive steps for training:
+
+  *  it applies a random projection to the input space, onto a possibly higher 
+     dimensional space;
+  *  the result passes through an element-wise non-linear activation function,
+     typically a Rectified Linear Unit or a sigmoid function (tanh or logistic); and
+  *  last, it trains a linear one vs. rest classifier or a multi-output ridge 
+     regression model.
+
+ELMs can be considered as single-hidden layer feedforward neural networks 
+(see figure 1) where:
+
+  *  they assign random values, as constants, to the input-to-hidden connection 
+     weights;
+  *  they only train the hidden-to-output connection weights; and
+  *  they minimize the loss function using least squares.
+
+In several empirical studies, ELM was found to have a generalization performance 
+similar to that of a one hidden layer multi-layer perceptron (MLP) or radial
+basis function (RBF) kernel support vector machines while being significantly 
+faster to train.
+
+.. figure:: ../auto_examples/neural_networks/images/plot_elm_diagram_001.png
+   :target: ../auto_examples/neural_networks/plot_elm_diagram.html
+   :align: center
+   :scale: 100%
+
+     Figure 1 : Single-hidden layer feedforward network.
+
+ELM has three main hyper-parameters,
+
+  *  ``weight_scale``: controls the variance of the random projection weights;
+  *  ``n_hidden``: controls the number of hidden layer nodes; and
+  *  ``C``: controls the regularization strength of the output linear model.
+
+``weight_scale`` and ``C`` are similar in that they regularize weight parameters
+such that a higher value means less regularization hence less underfitting (or less bias)
+but potentially more overfitting (or more variance). However, in terms of their
+effect on the single-hidden layer feedforward network, ``weight_scale`` regularizes 
+the input-to-hidden connection weights, whereas ``C`` regularizes the hidden-to-output 
+connection weights. Some examples demonstrating the effect of these two parameters are listed
+below:
+
+.. figure:: ../auto_examples/neural_networks/images/plot_elm_scale_C_and_weight_scale_001.png
+   :target: ../auto_examples/neural_networks/plot_elm_scale_C_and_weight_scale.html
```

Comment:
This seems very streched. I think you should adjust the aspect ratio of the plot.